dodiscover/context_builder.py

```diff
@@ -37,7 +58,47 @@ def graph(self, graph: Graph) -> "ContextBuilder":
         self : ContextBuilder
             The builder instance
         """
-        self._graph = graph
+        self._init_graph = graph
+        return self
+
+    def excluded_edges(self, exclude: Optional[NetworkxGraph]) -> "ContextBuilder":
+        """Set exclusion edge constraints to apply in discovery.
+
+        Parameters
+        ----------
+        excluded : Optional[NetworkxGraph]
+            Edges that should be excluded in the resultant graph
+
+        Returns
+        -------
+        self : ContextBuilder
+            The builder instance
+        """
+        if self._included_edges is not None:
+            for u, v in exclude.edges:  # type: ignore
+                if self._included_edges.has_edge(u, v):
+                    raise RuntimeError(f"{(u, v)} is already specified as an included edge.")
```

Comment:
or a warning?

Reply:
What would be the behavior then if we have both `(u, v)` as an included and excluded edge? Wouldn't this be a conflicting causal assumption?